### PR TITLE
EUI-6822 Two sections for the case file view

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-6822

### Change description ###
Case file view: Two sections for the case file view

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/shared/components/palette/palette.module.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ import { FormValidatorsService } from '../../services/form/form-validators.servi
 import { MarkdownModule } from '../markdown/markdown.module';
 import { AddressModule } from './address/address.module';
 import { BaseFieldModule } from './base-field/base-field.module';
+import { CaseFileViewModule } from './case-file-view';
```
Comment (Daniel-Lam) [inline src/shared/components/palette/palette.module.ts:20]: @johnbenjamin-hmcts - probably better to use the fully qualified form here, i.e. `'./case-file-view/case-file-view.module'`

File: src/shared/components/palette/palette.service.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { Injectable, Type } from '@angular/core';
 import { CaseField } from '../../domain/definition/case-field.model';
 import { WriteAddressFieldComponent } from './address/write-address-field.component';
+import { CaseFileViewFieldComponent } from './case-file-view';
```
Comment (Daniel-Lam) [inline src/shared/components/palette/palette.service.ts:4]: @johnbenjamin-hmcts - probably better to use the fully qualified form here, i.e. `'./case-file-view/case-file-view-field.component'`

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@johnbenjamin-hmcts - just two minor changes suggested; the rest is OK

File: src/shared/components/palette/palette.module.ts (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply src/shared/components/palette/palette.module.ts]: thank you @Daniel-Lam, I have now done the changes.

File: src/shared/components/palette/palette.service.ts (hunk 2 above)
Comment (johnbenjamin-hmcts) [reply src/shared/components/palette/palette.service.ts]: thank you @Daniel-Lam, I have now done the changes.

**Review by tarikdogan-hmcts — APPROVED** [review]

**Pull request merged**